source/MaterialXGenShader/HwShader.h

```diff
@@ -15,8 +15,7 @@ class HwShader : public Shader
 
 public:
     /// Identifier for additional vertex shader stage
-    static const size_t VERTEX_STAGE = Shader::NUM_STAGES;
-    static const size_t NUM_STAGES = Shader::NUM_STAGES + 1;
+    static const string VERTEX_STAGE;
```

Comment:
Using strings to identify shader stages instead of hard coded integers that need to stay in sync when deriving new stages.

Reply:
This is nicer. Thanks for the change.